index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
       <h2>I am a Software Developer</h2>
       <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
       <ul class="social">
-        <li><a href="https://github.com/Said-laasri">
-          <img src="img/github.svg" alt="github">
-        </a></li>
-        <li><a href="https://www.linkedin.com/in/said-laasri-8a4367172/">
-          <img src="img/linkdin.svg" alt="linkdin">
-        </a></li>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/win.svg" alt="win">
-        </a></li>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/twitter.svg" alt="twitter">
-        </a></li>
-        <li><a href="https://twitter.com/saidlaasr">
-          <img src="img/Vector.svg" alt="victor">
-        </a></li>
+        <li>
+          <a href="https://github.com/Said-laasri">
+            <img src="img/github.svg" alt="github">
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/said-laasri-8a4367172/">
+            <img src="img/linkdin.svg" alt="linkdin">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/win.svg" alt="win">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/twitter.svg" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/saidlaasr">
+            <img src="img/Vector.svg" alt="victor">
+          </a>
+        </li>
       </ul>
     </section>
+    <section class="work">
+      <h1 class="title2">My Recent Works</h1>
+      <section class="work project0">
+        <div class="info-project">
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <ul class="tags">
+          <li class="css">css</li>
+          <li class="html">html</li>
+          <li class="boot">bootstrap</li>
+          <li class="ruby">Ruby</li>
+        </ul>
+        <form class="check">
```

Comment:
- [ ] You can simply use the button without wrapping it with a form element because you are going to handle the button click effects later in the module using JavaScript. So, kindly remove the wrapping form element.

Reply:
done